src/pipe.js

```diff
@@ -24,15 +25,56 @@
         }
     }
 
+    function chunksCount(range) {
+        return (range[1] - range[0] + 1);
```

Comment:
This is only true if maxAssetLinks is 2.
E.g for `var range = [0,9]` it should return `10` and not `2` (`range.length`)